enterprise/internal/github_apps/store/store_test.go

```diff
@@ -400,9 +400,10 @@ func TestGetByDomain(t *testing.T) {
 
 	// does not exist
 	fetched, err = store.GetByDomain(ctx, &domain, "https://myCompany.github.com/")
+	_, ok := err.(ErrNoGitHubAppFound)
 	require.Nil(t, fetched)
 	require.Error(t, err)
-	require.ErrorContains(t, err, "no app exists matching criteria: {domain = %s AND base_url = %s [repos https://myCompany.github.com/]}")
+	require.Equal(t, ok, true)
```

Comment:
Could we also add keep an assertion about the contents of the error message here? Like:

```suggestion
	require.Equal(t, ok, true)
	require.Equal(t, err.Error(), "no app exists matching criteria: {WHERE domain = repos AND base_url = https://mycompany.github.com/}")
```

Or whatever it would be? :)

Reply:
Fixed this up :)